src/templates/default-page.js

```diff
@@ -16,13 +16,13 @@ export const DefaultPageTemplate = ({ seo, title, subTitle, content, contentComp
       {seo &&
         <Helmet title={seo.title ? seo.title : metadata.siteMetadata.title} titleTemplate={metadata.siteMetadata.titleTemplate}>
           {seo.description && <meta name="description" content={seo.description} />}
-          {seo.image && seo.url && <meta name="image" content={`${seo.url}${seo.image.publicURL}`} />}
+          {seo.image && seo.url && <meta name="image" content={`${seo.url}${seo.image}`} />}
```

Comment:
this is not a raw string please do check the CMS fie definition
this is breaking the upload funtionality should be seo.image.publicURL
https://github.com/OpenStackweb/ironic-website/blob/3918cd3ef6ff0d5bbab1347d3441ef0b0f088680/static/admin/config.yml#L124
if u dont see that field then u need to add the proper type def here
https://github.com/OpenStackweb/ironic-website/pull/73/files#r2349587881